chapter_generative-adversarial-networks/dcgan.md

```diff
@@ -109,14 +181,36 @@ net_G.add(G_block(n_G*8, strides=1, padding=0),  # Output: (64 * 8, 4, 4)
               activation='tanh'))  # Output: (3, 64, 64)
 ```
 
+```{.python .input}
+#@tab pytorch
+def Conv2DTranspose(channels, kernel_size, strides, padding, use_bias, nc=3):
+    return nn.ConvTranspose2d(nc, channels, kernel_size=kernel_size,stride=strides, padding=padding, bias=use_bias)
```

Comment:
What is the need to define this function?

Reply:
To look similar to mxnet.

Reply:
Just use the `nn.ConvTranspose2d` as is. No need to define a separate function with a different name doing exactly that.

Reply:
we need to know nc=3
I think it will be friendly to new users to show the similiarity and difference of pytorch and mxnet.

Reply:
But I followed your advice.
Look my newest PR.